docs/data/data-grid/export/export.md

```diff
@@ -6,46 +6,79 @@ title: Data Grid - Export
 
 {{"demo": "ExportDefaultToolbar.js", "bg": "inline"}}
 
-### Custom toolbar
+### Custom Toolbar
+
+The export menu is provided in a stand-alone component named `GridToolbarExport`. You can use it in a custom toolbar component as follow.
+
+```jsx
+function CustomToolbar() {
+  return (
+    <GridToolbarContainer>
+      <GridToolbarExport />
+    </GridToolbarContainer>
+  );
+}
+```
 
-To enable the CSV export in a custom toolbar, use the `GridToolbarExport` component.
+{{"demo": "ExportCustomToolbar.js", "bg": "inline", "defaultCodeOpen": false}}
 
-{{"demo": "ExportCustomToolbar.js", "bg": "inline"}}
+## Export options
 
-### Custom exported content
+By default, the export menu displays all the available export formats, according to your license, which are
 
-The csv export can be customized by passing a [`csvOptions`](/api/data-grid/grid-csv-export-options/) object either to the `GridToolbar` or to the `GridToolbarExport` as a prop.
+- [Print](#print-export)
+- [CSV](#csv-export)
+- [Excel](#excel-export) [<span class="plan-premium"></span>](https://mui.com/store/items/material-ui-pro/) (🚧 Not delivered yet)
+- [Clipboard](#clipboard) [<span class="plan-premium"></span>](https://mui.com/store/items/material-ui-pro/) (🚧 Not delivered yet)
 
-The following sections describes some customizations available on this interface.
+You can customize their respective behavior by passing an options object either to the `GridToolbar` or to the `GridToolbarExport` as a prop.
 
 ```tsx
 <DataGrid componentsProps={{ toolbar: { csvOptions } }} />
-
 // same as
-
 <GridToolbarExport csvOptions={csvOptions} />
 ```
 
-#### Exported columns
+Each export option has its own API page:
+
+- [`csvOptions`](/api/data-grid/grid-csv-export-options/)
+- [`printOptions`](/api/data-grid/grid-print-export-options/)
 
-By default, the CSV will only contain the visible columns of the grid.
-There are a few ways to include or hide other columns:
+## Disabled format
 
-1. Set the exact columns to be exported in [`csvOptions`](/api/data-grid/grid-csv-export-options/).
+You can remove an export format from the toolbar by setting its option property `disableToolbarButton` to `true`.
+In the following example, the print export is disabled.
 
 ```jsx
 <DataGrid
-  componentsProps={{ toolbar: { csvOptions: { fields: ['id', 'name'] } } }}
+  componentsProps={{ toolbar: { printOptions: { disableToolbarButton: true } } }}
 />
 ```
 
-2. Set `allColumns` in [`csvOptions`](/api/data-grid/grid-csv-export-options/) to `true` to include hidden columns, instead of only the visible ones.
+{{"demo": "RemovePrintExport.js", "bg": "inline", "defaultCodeOpen": false}}
+
+## Exported columns
+
+By default, the export will only contain the visible columns of the grid.
+There are a few ways to include or hide other columns. The exported columns are chosen in the following order:
```

Comment:
Since it is a bit tricky, maybe we can remove the notion of order which is different for print and csv. Merge as it is. We will re-write this section after #3904